### ISSUE TITLE
Parallel joyport adapter emulation broken?

### ISSUE THREAD

**Issue by Rinusch** [midwan/amiberry#977]
**Describe the bug**
I’m not able to control players 3 and 4 in 4 player games supporting the parallel port adapter.
Using the custom controls section, no matter what I map to the parallel port joysticks, triggers are not registered within the emulated Amiga. So whether I launch a game supporting the parallel port joysticks, or a joystick testing utility supporting them...they won't register the triggers.

**To Reproduce**
- Go to tab 'input' and set any input controller for ports 2 and 3.
- Go to tab 'custom input' and map any button(s) you can (and want) to the amiga parallel joystick 1 and 2. You can for instance only map a single button to parallel joy 1 up which should suffice for reproduction.
- Start a game supporting the parallel joystick ports (for support of more than 2 players), e.g. Dynablaster, or a joystick testing utility. (note: in case of Dynablaster, the WHDload script can be used to select 2 types of emulated adapters. Both have different behaviour, but none will register triggers. You should select battle mode with 5 players to be sure the performed mapping should at least control one of the players)
- In game or in joystick testing utility, use the mapped button from the selected controller and notice nothing happens.

**Expected behavior**
Players 3 and 4 of any game supporting the parallel joystick adapter should respond to the mapped customized control.

**Additional context**
Mapping for instance an input to the Amiga keyboard does work, this issue is purely related to the **parallel port joysticks**.
I'm using Amiberry 3.3 (and 5.2) on Retropie. Both versions have the issue.

**Comment by midwan** [comment]
You don't normally need to map any custom controls to the Parallel ports, in order for them to work.
You should be able to assign a controller there, and the mapping of that controller will be used on that port automatically.

I was able to test this with the current version (5.2) and play Dynablaster and Gauntlet II with 4 players, without problems.

Why are you trying to use custorm controls on the parallel port joysticks?

**Comment by Rinusch** [comment]
Hi!

I'm sorry not having responded sooner. I wanted to update my system first, in order to test with the 5.2 version you mentioned.
I upgrade Retropie first, including all packages (including Amiberry). So I went from Amiberry 3.3 to 5.2 and it works now! Has anything changed from 3.3 to 5.2 in terms of parallel joysticks support?

The thing is, I briefly tested it on 5.2 before on another Pi system and it didn't work. Not sure why that was but I must have done something wrong there, maybe only testing with Dynablaster and with the wrong configuration of the WHDLoad script on startup of the game.

Anyway, I'm VERY happy it's working now. Thanks for your help and great work!

**Comment by midwan** [comment]
@Rinusch 
Yes, this was a bug that was fixed quite a while ago (before 5.2). Good to see that it works for you now! :)
